man/systemd-analyze.xml

```diff
@@ -42,7 +42,7 @@
     <cmdsynopsis>
       <command>systemd-analyze</command>
       <arg choice="opt" rep="repeat">OPTIONS</arg>
-      <arg choice="plain">dump</arg>
+      <arg choice="plain">dump<replaceable>PATTERN</replaceable></arg>
```

Comment:
Please do not append like that, but insert a new line with
`<arg choice="opt" rep="repeat"><replaceable>PATTERN</replaceable></arg>`.